usage/src/main/java/org/killbill/billing/usage/plugin/api/UsagePluginApi.java

```diff
@@ -21,12 +21,12 @@
 import java.util.UUID;
 
 import org.joda.time.LocalDate;
+import org.killbill.billing.payment.api.PluginProperty;
 import org.killbill.billing.usage.api.RawUsageRecord;
-import org.killbill.billing.util.callcontext.TenantContext;
 
 public interface UsagePluginApi {
 
-    List<RawUsageRecord> getUsageForAccount(final LocalDate startDate, final LocalDate endDate, final TenantContext context);
+    List<RawUsageRecord> getUsageForAccount(final LocalDate startDate, final LocalDate endDate, final UsageContext context, Iterable<PluginProperty> properties);
```

Comment:
Nit: either `final` throughout or none at all.

Reply:
Fixed in https://github.com/killbill/killbill-plugin-api/pull/21/commits/588163af3a06752b46ef448f1c76519b16f9a790